dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/core/process_definition.py

```diff
@@ -381,6 +400,7 @@ def submit(self) -> int:
             self.timeout,
             self.worker_group,
             self._tenant,
+            self.release_state,
```

Comment:
It should be `self._release_state`, because we want to covert the input string `online` or `offline` to output int `0` or `1`. So we have a `property` named `release_state` to do it